src/config/frontend/EdFi.DmsConfigurationService.Frontend.AspNetCore/Infrastructure/SecurityConstants.cs

```diff
@@ -8,4 +8,5 @@ namespace EdFi.DmsConfigurationService.Frontend.AspNetCore.Infrastructure;
 public static class SecurityConstants
 {
     public const string ServicePolicy = "ServicePolicy";
+    public const string ScopeFullAccess = "scp:edfi_dms_configuration_service/full_access";
```

Comment:
I kind of just made this up based on what I saw in AdminApi.

Reply:
@simpat-adam 
We lose the flexibility of configuring the custom scope name manually. Wouldn't it be better if we could simply read it from the appsettings configuration?